cmd/av/stack_submit.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/config"
+	"github.com/aviator-co/av/internal/gh"
+	"github.com/aviator-co/av/internal/meta"
+	"github.com/shurcooL/githubv4"
+	"github.com/spf13/cobra"
+)
+
+var stackSubmitCmd = &cobra.Command{
+	Use:   "submit",
+	Short: "create pull requests for the current stack",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) > 0 {
+			_ = cmd.Usage()
+			return errors.New("this command takes no arguments")
+		}
+
+		// Get the all branches in the stack
+		repo, _, err := getRepoInfo()
+		if err != nil {
+			return err
+		}
+		branches, err := meta.ReadAllBranches(repo)
```

Comment:
`ReadAllBranches` returns every branch that `av` knows about, not just the branches in the current stack.